### PR TITLE
Che ping/bidding page dr

### PR DESCRIPTION
**Pull request by curry0622**
# Bidding Page DR
1. Graph
2. Dr
## Graph
* Displays a bar chart when current mode is `需量反應`, depends on which day is it (7 different data in total).
## Dr
* Display based on fake data
* Edit bid and add bid
* Still unable to switch pages, all in one page now
* RWD

### CODE REVIEW

File: src/components/biddingPage/dr/addBid.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        time,
+        value,
+        price,
+        total,
+        status: 'new',
+        accepted: false,
+      });
+      // tmpDataArr.sort((a, b) => {
+      //   return new Date(a.date).getTime() - new Date(b.date).getTime();
+      // });
```
Comment (coliver0221) [inline src/components/biddingPage/dr/addBid.tsx]: Please remove unnecessary comments.

File: src/components/biddingPage/graph/barChart.tsx [hunk 2]
```diff
@@ -1,8 +1,13 @@
 import React, { useEffect, useRef, useState } from 'react';
 import * as d3 from 'd3';
 import classNames from 'classnames';
-import data from './test.json';
+// import data from './test.json';
```
Comment (coliver0221) [inline src/components/biddingPage/graph/barChart.tsx]: Please remove useless import.

File: src/components/biddingPage/graph/barChart.tsx [hunk 3]
```diff
@@ -71,6 +82,20 @@ const BarChart: React.FC = () => {
...
+    setDay(new Date(date).getDay());
+  }, [date]);
+
+  // set the data to display depends on day
+  useEffect(() => {
+    weekData.map((d) => {
+      if (d.day === day + 1) setData(d.data);
+      return null;
+    });
+    // alert(day);
```
Comment (coliver0221) [inline src/components/biddingPage/graph/barChart.tsx]: Please remove unnecessary comment.

Comment (curry0622) [reply src/components/biddingPage/graph/barChart.tsx]: Done. Thanks

File: src/components/biddingPage/graph/barChart.tsx (hunk 2 above)
Comment (curry0622) [reply src/components/biddingPage/graph/barChart.tsx]: Done. Thanks

File: src/components/biddingPage/dr/addBid.tsx (hunk 1 above)
Comment (curry0622) [reply src/components/biddingPage/dr/addBid.tsx]: Done. Thanks

**Comment by coliver0221** [comment]
LGTM.

**Pull request merged**